src/expressServer.js

```diff
@@ -18,11 +18,9 @@ const { readFileSync } = require('fs');
 const { join } = require('path');
 const expressStaticGzip = require('express-static-gzip');
 const recommendationRoutes = require('./routes/recommendationRoutes');
+const imgur = require('./views/imgur/imgur');
 
 const app = express();
-// Express Async Errors must be used before any route is used,
-// it will catch any errors that are thrown in the async functions without needing a try/catch
-require('express-async-errors');
```

Comment:
Any reason for deleting the comment + moving this line? 

It's fine since it doesn't change any behaviour of the app but just curious if it was an intentional change

Reply:
No, it wasn't intentional change, corrected commit 9cb630ca70d7b497bcbc7be339804d0192b7a2cc